radixdlt-core/radixdlt/src/main/java/com/radixdlt/api/system/prometheus/PrometheusService.java

```diff
@@ -223,6 +234,26 @@ private void addBranchAndCommit(StringBuilder builder) {
     appendField(builder, "branch_and_commit", branchAndCommit);
   }
 
+  private void addCurrentFork(StringBuilder builder) {
+    appendField(builder, "current_fork_name", currentForkView.currentForkConfig().name());
+  }
+
+  private void addCandidateForkRemainingEpochs(StringBuilder builder) {
+    engineStatusService
+        .getCandidateForkRemainingEpochs()
+        .ifPresent(
+            candidateForkRemainingEpochs ->
+                appendField(
+                    builder, "candidate_fork_remaining_epochs", candidateForkRemainingEpochs));
+  }
+
+  private void addCandidateForkVotingResult(StringBuilder builder) {
+    appendField(
+        builder,
+        "candidate_fork_voting_result",
+        engineStatusService.getCandidateForkVotingResult());
```

Comment:
Prometheus metrics are floating point, so can we make this a percentage?

And I guess call it something like `candidate_fork_voting_result_percentage`?